### PR TITLE
README: fix launched chrome example

### PR DESCRIPTION
**Pull request by ebidel**
Also removes calc'ing overall score based on old aggregations.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -160,16 +160,15 @@ assumes you've installed Lighthouse as a dependency (`yarn add --dev lighthouse`
...
 const Printer = require('lighthouse/lighthouse-cli/printer');
 
 function launchChromeAndRunLighthouse(url, flags, config) {
   const launchedChrome = ChromeLauncher.launch();
 
   launchedChrome
     .then(() => lighthouse(url, flags, config)) // Run Lighthouse.
-    .then(results => launchedChrome.kill())
-    .then(() => results) // Kill Chrome and return results.
+    .then(results => launchedChrome.kill().then(() => results)) // Kill Chrome and return results.
```
Comment (stramel) [inline readme.md]: Isn't this an anti-pattern? Wouldn't keeping the promise chain flat be better?

**Comment by samccone** [comment]
oops

File: readme.md (hunk 1 above)
Comment (ebidel) [inline readme.md]: `results` isn't in scope. You could stash a var, but I actually think that complicates the example.

Comment (stramel) [inline readme.md]: Ah, yeah you would have to return results from it to use it on the next iteration.. gotcha.

**Pull request merged**